picard/util/tags.py

```diff
@@ -92,6 +92,15 @@
     'musicbrainz_originalartistid': N_('MusicBrainz Original Artist Id'),
     'originalalbum': N_('Original Album'),
     'musicbrainz_originalalbumid': N_('MusicBrainz Original Release Id'),
+    'movementname': N_('Movement Name (iTunes Classical)'),
+    'musicbrainz_movementid': N_('MusicBrainz Movement Work Id'),
+    'movementnumber': N_('Movement Number (iTunes Classical)'),
+    'movementtotal': N_('Movement Total (iTunes Classical)'),
+    'itunesgrouping': N_('Grouping (iTunes Classical)'), # ID3 GRP1 introduced in iTunes 12.5.4.42 Dec 2016
+    'ituneswork': N_('Work (iTunes Classical)'), # mp4 ©wrk introduced in iTunes 2016
+    '~movementshow': N_('Movement Show (iTunes Classical)'),
+    '~workcomment': N_('Work Disambiguation'),
+    '~movementcomment': N_('Movement Disambiguation (iTunes Classical)'),
```

Comment:
I can never remember if `~` tags are not written to files for all formats, or if id3 writes them under some circumstances, but if they're not written, how can this be an `iTunes Classical` tag, if it's only used internally by picard?

Reply:
It is data available from iTunes but there are no tags to map it to. So it is not written to any formats unless user provides a tagging script to copy it to a variable which will end up as an e.g. TXXX tag.